Formula/resty.rb

```diff
@@ -1,29 +1,57 @@
 class Resty < Formula
   desc "Command-line REST client that can be used in pipelines"
   homepage "https://github.com/micha/resty"
-  url "https://github.com/micha/resty/archive/2.2.tar.gz"
-  sha256 "a6961e16f1489ffa25bf9f2d8b38b9599c008d49267e32adf93f4f87184d4857"
+  url "https://github.com/micha/resty/archive/2.3.tar.gz"
+  sha256 "65c7bfe1268669d9da5456686c55f1976380b867db5bf12d3603972d7b2d65fd"
   head "https://github.com/micha/resty.git"
 
-  bottle :unneeded
-
   # Don't take +x off these files
   skip_clean "bin"
 
+  resource "JSON" do
+    url "https://cpan.metacpan.org/authors/id/I/IS/ISHIGAKI/JSON-2.94.tar.gz"
+    mirror "http://search.cpan.org/CPAN/authors/id/I/IS/ISHIGAKI/JSON-2.94.tar.gz"
+    sha256 "12271b5cee49943bbdde430eef58f1fe64ba6561980b22c69585e08fc977dc6d"
+  end
+
   def install
-    bin.install %w[pp resty pypp]
+    pkgshare.install "resty"
+
+    ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
+
+    resource("JSON").stage do
+      system "perl", "Makefile.PL", "INSTALL_BASE=#{libexec}"
+      system "make"
+      system "make", "install"
+    end
+
+    bin.install "pp"
+    bin.env_script_all_files(libexec/"bin", :PERL5LIB => ENV["PERL5LIB"])
+    chmod 0755, "#{bin}/pp"  # note: to be killed once a shebang is added to pp
```

Comment:
note the immediate issue is actually with pypp not pp but yes they both should have a shebang